.github/workflows/publish-release-container.yml

```diff
@@ -0,0 +1,45 @@
+name: Build and Publish container for new releases
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    name:                         Build and push image
+    runs-on:                      ubuntu-latest
+    steps:
+
+    - uses:                       actions/checkout@v2
+
+    - name: Workaround rootless build
+    # temporary workaround for https://github.com/redhat-actions/buildah-build/issues/45
+      run: |
+        sudo apt-get install fuse-overlayfs
+        mkdir -vp ~/.config/containers
+        printf "[storage.options]\nmount_program=\"/usr/bin/fuse-overlayfs\"" > ~/.config/containers/storage.conf
```

Comment:
```suggestion
```
As I mentioned, please refer to a fixed job https://github.com/paritytech/substrate-api-sidecar/blob/master/.github/workflows/publish-release-container.yml

Reply:
not sure i'm following this comment. What do you mean that i have to refer to a fixed job? Is this just a matter of remove that part of the code or should i move this code to the `js.yml`?

Reply:
just apply the suggestion